.eslintrc

```diff
@@ -13,7 +13,7 @@
     },
     {
       "files": ["**/*.{ts,tsx}"],
-      "globals": { "globalThis": false }
+      "globals": { "globalThis": false, "JSX": true }
```

Comment:
I think the "jsx: true" rule should be on the `"files": ["**/*.tsx"],` rule not this one.